controllers/k8ssandra/reaper.go

```diff
@@ -290,3 +314,69 @@ func (r *K8ssandraClusterReconciler) addClusterToExternalReaper(
 	}
 	return result.Continue()
 }
+
+func addManagementApiSecretsToReaper(
+	ctx context.Context,
+	remoteClient client.Client,
+	kc *api.K8ssandraCluster,
+	actualDc *cassdcapi.CassandraDatacenter,
+	logger logr.Logger,
+	authConfig *cassdcapi.ManagementApiAuthManualConfig,
+) error {
+
+	reaperName := kc.Spec.Reaper.ReaperRef.Name
+
+	reaperNamespace := kc.Spec.Reaper.ReaperRef.Namespace
+	if reaperNamespace == "" {
+		reaperNamespace = kc.Namespace
```

Comment:
Question: Is this the right default? It seems to me like by default Reaper will deploy to the mission control namespace as a static resource. But I kind of prefer this logic because it mirrors what we do elsewhere more closely...

Reply:
It is indeed very likely we'll be deploying Reaper in the `mission-control` namespace. But I also think we should not be hardcoing the defaults of that namespace anywhere. I think it once bit me already, althought I don't remember where.
So I went for waht we do elsewhere. We seem to be in agreement, so I'll go ahead and resolve this bit.